routers/api/v1/repo/release_attachment.go

```diff
@@ -35,6 +35,50 @@ func GetReleaseAttachment(ctx *context.APIContext) {
 	//   description: name of the repo
 	//   type: string
 	//   required: true
+	// - name: attachment_id
+	//   in: path
+	//   description: id of the attachment to get
+	//   type: integer
+	//   format: int64
+	//   required: true
+	// responses:
+	//   "200":
+	//     "$ref": "#/responses/Attachment"
+
+	attachID := ctx.ParamsInt64(":asset")
+	attach, err := models.GetAttachmentByID(attachID)
+	if err != nil {
+		ctx.Error(http.StatusInternalServerError, "GetAttachmentByID", err)
+		return
+	}
```

Comment:
with this i can delete attachments i should not have the rights to do so ... you have to test if attachment you get belongs to this repo ...

Reply:
& tests for this are needed + some general tests too